packages/local-cli/core/__fixtures__/projects.js

```diff
@@ -34,4 +34,4 @@ const withPods = {
   ios: ios.pod,
 };
 
-module.exports = {flat, nested, withExamples, withPods};
```

Comment:
so we go with non-fb style of extra spacing in destructuring statements?

Reply:
Back to the roots! https://github.com/rnpm/rnpm/blob/master/src/config/android/index.js#L110

On a serious note: I don't have preference here. Just sticking to whatever ESLint preset is recommending. I guess we can always change that in case we agree on a general preset for all RNCommunity projects - right now, it's pretty much up to project maintainers.